doc/fuzzing.md

```diff
@@ -26,7 +26,7 @@ To build Bitcoin Core using AFL instrumentation (this assumes that the
 ./configure --disable-ccache --disable-shared --enable-tests CC=${AFLPATH}/afl-gcc CXX=${AFLPATH}/afl-g++
 export AFL_HARDEN=1
 cd src/
-make test/test_bitcoin_fuzzy
+make -C src/test
```

Comment:
User is already in the `src` directory so this will fail

Reply:
Done (changed it to `make`)